docker-app/qfieldcloud/core/utils2/sentry.py

```diff
@@ -0,0 +1,50 @@
+import logging
```

Comment:
One more question, could we do without this threading code?

We transfer a few couple of MB upon a failed request, which should IMO be fine to do on the main thread and ideally we could avoid a lot of boilerplate code in here.

Reply:
Done!